pyvrp/educate/tests/test_MoveTwoClientsReversed.py

```diff
@@ -33,13 +33,14 @@ def test_single_route_OkSmall():
     assert_equal(individual.num_routes(), 1)
     assert_(individual.cost() < copy.cost())
 
-    # (2, 3) was inserted after the depot as (3, 2)
-    assert_equal(individual.get_routes(), [[3, 2, 1, 4], [], []])
+    # (2, 3) was inserted after 1 as 1 -> 3 -> 2 -> 4. Then (1, 3) got inserted
+    # after 2 as 2 -> 3 -> 1 -> 4.
+    assert_equal(individual.get_routes(), [[2, 3, 1, 4], [], []])
```

Comment:
Was this changed as a result of the bugfix? Do we still need to have the tests below as we test for this exact solution?

Reply:
Yes to the first part. What do you mean by the second question?